src/sentry/tasks/derive_code_mappings.py

```diff
@@ -29,15 +29,19 @@ def identify_stacktrace_paths(
     Generate a map of projects to stacktrace paths for specified organizations,
     or all active organizations if unspecified.
 
-    This filters out projects have not had an event in the last 7 days or have
-    non-python files in the stacktrace.
+    This filters out non-python projects, or projects without an event
+    in the last 7 days.
     """
     if organizations is None:
         organizations = Organization.objects.filter(status=OrganizationStatus.ACTIVE)
 
     filename_maps = {}
     for org in organizations:
-        projects = Project.objects.filter(organization=org, first_event__isnull=False)
+        projects = Project.objects.filter(
+            organization=org,
+            first_event__isnull=False,
+            platform="python",
```

Comment:
Is this the platform set by the customer on project creation? If that's the case we don't want this. I may have misunderstood your comment in Slack. I thought we were going to rely on the SDK's platform.

I assume a Group or the events may have a platform associated to it based from the SDK.

Reply:
Ah good point, missed that distinction.